src/database/cloudant.js

```diff
@@ -21,17 +21,20 @@ class CloudantDatabase {
   createPartition(partitionKey, database) {
     // https://www.npmjs.com/package/@cloudant/cloudant - cloudant library with example db operations
     const partition = {
-      async create(data) {
+      async create(data, insertId = false) {
         return new Promise((resolve, reject) => {
           const id = uuidv4();
           const _id = `${partitionKey}:${id}`;
           const documentWithId = { ...data, _id };
-          database.insert(documentWithId, (err, document) => {
+          if (insertId) {
+            documentWithId.id = id;
```

Comment:
Is this necessary? If your resolver needs it could just use parent._id

Reply:
I thought about it and I think insertId should be true by default. The reason why we need this is because "id" and "_id" are two different values. "_id" is "partitionKey:id" whereas id is just the id